module-new/src/components/switch/switch.theme.css

```diff
@@ -0,0 +1,85 @@
+/* CONTAINER */
+
+.arm-switch-container {
+  display: flex;
+  align-items: center;
+}
+
+.arm-switch-container[data-disabled='true'] {
+  filter: none;
+  opacity: 0.6;
+}
+
+.arm-switch-container[data-direction='vertical'] {
```

Comment:
there is no vertical mode in the designs - if it just switching flex direction, is it necessary to have in armstrong vs just adding a custom class name in the consuming code?

Reply:
Fair one - it was present in the previous version of the component so wasn't sure what to migrate. Happy to remove and do the same for the switch for readability/simplicity @jlews ?

Reply:
I vote for remove @tomRocketmakers in favour of simplicity, the consuming app can always override the classes for edge cases